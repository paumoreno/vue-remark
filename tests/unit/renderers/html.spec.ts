import VueRemarkHtml from "@/renderers/html.vue";
import { mount } from "@vue/test-utils";

describe(VueRemarkHtml.name, () => {
  test("snapshot", () => {
    const wrapper = mount(VueRemarkHtml, {
      propsData: {
        position: {
          start: {
            line: 1
          },
          end: {
            line: 1
          }
        },
        value: `<h1>html heading</h1>`
      }
    });

    expect(wrapper).toMatchSnapshot();
  });
});
