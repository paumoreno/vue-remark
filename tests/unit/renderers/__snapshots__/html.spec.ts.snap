// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VueRemarkHtml snapshot 1`] = `
<div>
  <h1>html heading</h1>
</div>
`;
